lib/oli_web/templates/layout/_delivery_header.html.eex

```diff
@@ -39,3 +39,12 @@
     <% end %>
   </div>
 </nav>
+
+<%= if (Map.has_key?(@conn.assigns, :delivery_breadcrumb) and @delivery_breadcrumb)
```

Comment:
This feels complicated enough to perhaps move into view helper/function, what do you think?